learn/experimental/overview.mdx

```diff
@@ -0,0 +1,40 @@
+# Experimental features
+
+Meilisearch periodically introduces new experimental features. Experimental features are not yet ready for production, but offer functionality that might benefit some users.
```

Comment:
Would you say the same about `--experimental-reduce-indexing-memory-usage`, @dureuill? My understanding was that this feature was tagged as experimental not because we're unsure about stabilising it, but because we'd like to see it in action before integrating it in our codebase.